README.md

```diff
@@ -2,6 +2,160 @@
 
 Command-line interface for Jetfire/Transformations.
 
+## Usage
+
+### Authenticate
+
+To use `jetfire-cli`, the `JETFIRE_API_KEY` environment variable must be set to a valid API key for a service account which has access to Jetfire/Transformations.
+
+By default, `jetfire-cli` runs against the main CDF cluster (europe-west1-1).
+To use a different cluster, specify the `--cluster` parameter. Note that this is a global parameter, which must be specified before the subcommand.
```

Comment:
It might be better to call this `--baseUrl` as we do for other tooling and allow the user to specify `https://...`, as there are also use cases (statnett) where they might need to use a proxy and go through `http://127.0.0.1` or whatever.

Reply:
Also for weird Azure or on-prem deployments that may not have a `cognitedata.com` URL.

Reply:
Yeah, I agree. The idea behind this was to ease the transition from `https://jetfire.*.cogniteapp.com/` to `*.cognitedata.com` when that day comes. But I did not consider proxies, azure, or testing. There is currently only a single user, so changing this should be trivial. Perhaps we can support either-or? Not sure.

Reply:
In the long run I think it's better if we standardize on baseUrl everywhere.

Reply:
Agreed.